chore(obolo_chwerty): broken links in help

It looks like there are some files missing from help for obolo_chwerty 1.2.1:

```
Getting links from: http://localhost:8888/keyboard/obolo_chwerty
├─BROKEN─ http://localhost:8888/keyboard/obolo_chwerty/1.2.1/chwertyenterkey1.png (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/obolo_chwerty/1.2.1/chwertyenterkey2.png (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/obolo_chwerty/1.2.1/chwertyenterkey3.png (HTTP_404)
Error: Process completed with exit code 1.
```

--- a/release/o/obolo_chwerty/source/help/obolo_chwerty.php
+++ b/release/o/obolo_chwerty/source/help/obolo_chwerty.php
@@ -1,14 +1,14 @@
-﻿<?php 
+﻿<?php
  $pagename = 'Obolo Chwerty Keyboard Help';
  $pagetitle = $pagename;
  require_once('header.php');
 ?>
-<p>Obolo Chwerty is a simple Unicode keyboard for the Obolo language, using the CHWERTY layout <samp>ch</samp>, <samp>ọ</samp>, and <samp>n̄</samp>.</p> 
+<p>Obolo Chwerty is a simple Unicode keyboard for the Obolo language, using the CHWERTY layout <samp>ch</samp>, <samp>ọ</samp>, and <samp>n̄</samp>.</p>
 <p>The keyboard uses CHWERTY layout.
 <p>
 "Ebi Obolo" do not have 'C' and 'H' in their "abiche" (alphabet), but they have the 'CH' sound (as well as the 'SH' sound in some dialect). This and other features make Obolo Language unique. Hence this unique layout.
 <p>
-<!-- Insert Keyboard Layout Images or HTML here --> 
+<!-- Insert Keyboard Layout Images or HTML here -->
 <img src="chwerty_default.png"
 alt="Keyboard Layout (default)"
 style="width:695px;height:189px;">
@@ -21,9 +21,9 @@ style="width:695px;height:189px;">
  <p>(1) You cannot place more than one accent on a letter.
  <p>(2) Some accents are optional and are not required. They are only used in primers and for beginners.
  <p>(3) Some letters do not accept some accents.
- <p>(4) Some letters do not accept accents at all. 
-<p> 
-Except for the optional accents which are not required, the keyboard will notify you when you inadvertently go against the spelling rules. 
+ <p>(4) Some letters do not accept accents at all.
+<p>
+Except for the optional accents which are not required, the keyboard will notify you when you inadvertently go against the spelling rules.
 <p>
 <h4>Function</h4>
 <p>The Obolo Chwerty keyboard follows the English (QWERTY US) layout. On computers, you can type all of the special Obolo letters with a single keystroke using this keyboard. Here is how:</p>
@@ -46,18 +46,18 @@ The position of your BACKSLASH key depends on type of keyboard you are using.
 <p>The three keyboard types and the respective position of the BACKSLASH keys are shown below.
 <p>You can tell where your BACKSLASH key is by looking at the shape of your ENTER/RETURN key:
 <p>
- <img src="chwertyenterkey1.png"
+ <img src="chwertyftkey1.png"
 alt="ANSI Keyboard">
 <p>
-<img src="chwertyenterkey2.png"
+<img src="chwertyftkey2.png"
 alt="ISO/JIS Keyboard">
 <p>
-<img src="chwertyenterkey3.png"
+<img src="chwertyftkey3.png"
 alt="Backward-L Keyboard">
 <p>
 Keyman ... type to the world in your language.
 <p>
 Obolo Chwerty ... ge inu me usem kwun̄ ria si ere geelek me linyọn̄.
-<p>Onirọ-o! </p> 
+<p>Onirọ-o! </p>
 <p>
 Itọn̄ ikup inyi Awaji!</p>

